t/form.t

```diff
@@ -543,24 +543,24 @@ $f = HTML::Form->parse( <<EOT, "http://www.example.com" );
   <table>
     <TR>
       <TD align="left" colspan="2">
-	&nbsp;&nbsp;&nbsp;&nbsp;Keep me informed on the progress of this election
-	<INPUT type="checkbox" id="keep_informed" name="keep_informed" value="yes" checked>
+  &nbsp;&nbsp;&nbsp;&nbsp;Keep me informed on the progress of this election
+  <INPUT type="checkbox" id="keep_informed" name="keep_informed" value="yes" checked>
       </TD>
```

Comment:
Not sure that it matters, but the alignment got messed up here. I'm guess it was tabs before?

Reply:
Yes, it was. That was the only thing that failed the tabs test. I converted it, and it looked like it didn't change the visual indentation, but I guess I was wrong. Thanks for pointing it out.